package-lock.json

```diff
@@ -42530,7 +42530,7 @@
 			"dependencies": {
 				"clone-deep": {
 					"version": "0.2.4",
-					"resolved": "https://registry.npmjs.org/clone-deep/-/clone-deep-0.2.4.tgz",
+					"resolved": "http://registry.npmjs.org/clone-deep/-/clone-deep-0.2.4.tgz",
```

Comment:
That's weird, it should be https?

Reply:
Yes, it's weird. I removed `node_modules` and executed `npm i` twice and it still didn't update. I will revert and see if it stays this way.